server/src/migration/1673454659332-CreateTables.ts

```diff
@@ -0,0 +1,81 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class CreateTables1673454659332 implements MigrationInterface {
```

Comment:
This file is not needed! Looks great otherwise!

Reply:
Deleted, thank you!